### PR TITLE
QOL-7740 prepare for CKAN 2.9

### PR DESCRIPTION
**Pull request by ThrawnCA**
Add selenium tests for additions to resource form.

This should more easily handle changes to the underlying implementation.

### CODE REVIEW

File: .docker/scripts/create-test-data.sh [hunk 1]
```diff
@@ -63,7 +113,7 @@ echo ${organisation_admin_update}
 echo "Updating publisher to have editor privileges in the department-of-health Organisation:"
 publisher_update=$( \
     curl -LsH "Authorization: ${API_KEY}" \
-    --data "id=department-of-health&username=publisher&role=editor" \
+    --data "id=department-of-health&username=editor&role=editor" \
```
Comment (duttonw) [inline .docker/scripts/create-test-data.sh:116]: should this be in TEST_ORG_NAME also?

File: test/features/resource_validation.feature [hunk 2]
```diff
@@ -0,0 +1,136 @@
+Feature: Resource validation
+
+    Scenario: As an editor, I can create and update a resource with an uploaded validation schema
```
Comment (duttonw) [inline test/features/resource_validation.feature:3]: can you add scenario outlines/ examples table to group some tests together
https://behave.readthedocs.io/en/latest/tutorial.html#scenario-outlines

File: .docker/scripts/create-test-data.sh (hunk 1 above)
Comment (ThrawnCA) [reply .docker/scripts/create-test-data.sh:116]: I don't think we're using any tests where it matters, yet. This just keeps it in sync with ckanext-qgov.

File: test/features/resource_validation.feature (hunk 2 above)
Comment (ThrawnCA) [reply test/features/resource_validation.feature:3]: It's possible, but there's too much different behaviour between the different scenarios for it to really be worthwhile, I think. I could combine the validation schema and validation options scenarios into one, if you want, but it still wouldn't be an outline.

**Review by duttonw — APPROVED** [review]

**Pull request merged**